src/plugins/lite/plugin.ts

```diff
@@ -1085,20 +1086,43 @@ LITEPlugin.prototype = {
     return this.rejectChange(node);
   },
 
+  /**
+   * Reverts a change element (ins/del) using the revert-info attribute, which contains a serialized
+   * form of the previous change elements attributes.
+   */
+  _revertFromInfo: function(changeElement: any) {
+    // find the revert info attribute which contains a serialized version of the previous change
+    const revertInfo = changeElement.getAttribute(LITEConstants.attributes.revertInfo);
+    if (!!revertInfo) {
+      let attrs;
+      try {
+        // parse the revert-info JSON into a js object
+        attrs = JSON.parse(revertInfo);
+      } catch (e) {
+        console.warn('Could not parse akorda revert information:', revertInfo);
+      }
+      if (!!attrs) {
+        // Create a new change element that is the inverse of the current change element
+        const changeType = this._isInsert(changeElement) ? 'deleteType' : 'insertType';
+        const revertToElement = new CKEDITOR.dom.element(this._tracker._createIceNode(changeType));
+        // copy over the html from the delete and set the attributes
+        revertToElement.setHtml(changeElement.getHtml());
+        revertToElement.setAttributes(attrs);
+        // add the new ins to the dom and delete the old del
+        revertToElement.insertBefore(changeElement);
+        changeElement.remove();
```

Comment:
For browser compatibility I think is better to use removeChild from its parentNode

Reply:
This is a CKEditor element, so we're using editor's element.remove() API method here.